src/components/deploymentConfig/deploymentConfig.scss

```diff
@@ -508,3 +508,11 @@
         stroke: #fff;
     }
 }
+
+.position-bottom {
+    z-index: 21;
```

Comment:
Can we reduce this?

Reply:
not below this since by default Drawer has z-index: 20, and I want on top of that